src/components/SettingsView/RandomSeedInput.js

```diff
@@ -0,0 +1,52 @@
+import React, { useContext, useState } from 'react'
+import { string } from 'prop-types'
+import window from 'global/window'
+import TextField from '@material-ui/core/TextField'
+
+import FarmhandContext from '../Farmhand/Farmhand.context'
+
+import './RandomSeedInput.sass'
+
+export const RandomSeedInput = ({ search = window.location.search }) => {
```

Comment:
does this one have to worry about `window` not existing?

Reply:
Nope, it's imported up on line 3. It's not strictly necessary, but I've found using the `global/window` module to be a helpful practice, especially for test mocks.

Reply:
ohhh, i see what you did now, i like it. nice one